Source/Widgets/CesiumInspector/CesiumInspector.js

```diff
@@ -112,15 +112,15 @@ define([
         shaderCacheDisplay.className = 'cesium-cesiumInspector-shaderCache';
         shaderCacheDisplay.setAttribute('data-bind', 'html: shaderCacheText');
         generalSection.appendChild(shaderCacheDisplay);
-
+/*
```

Comment:
1. We always use single line quotes instead of block quotes.
2. Add a link to the issue that is causing us to temporarily disable these (though wouldn't it be better to remove the code altogether and just add it back later?)